docs/build_docs.py

```diff
@@ -0,0 +1,880 @@
+        )
+        f.write(release_note_content)
+
+
+def build_release_notes_with_brassy(
+    releases_dir, license_url, log=logging.getLogger(__name__)
+):
+    """Generate release notes for each subdirectory in a specified releases directory.
+
+    Uses the `brassy` build tool and the `pinkrst` formatting tool to build release
+    notes with a license disclaimer at the top.
+
+    Parameters
+    ----------
+    releases_dir : str
+        The path to the main directory containing individual release directories.
+    license_url : str
+        The URL pointing to the license or distribution statement for the release notes.
+    log : logging.Logger, optional
+        Logger instance used for logging debug and warning messages. By default,
+        uses a logger with the module's name.
+
+    Notes
+    -----
+    Each subdirectory in `releases_dir` is assumed to correspond to a release version.
+    This function generates a header file containing a distribution statement that
+    includes `license_url`. For each release directory, an `.rst` file with the
+    release notes is created and processed by `build_release_note_from_dir_with_brassy`.
+    The function will ignore directories named "upcoming" and log a warning.
+
+    Warnings
+    --------
+    Directories in `releases_dir` named "upcoming" are skipped, and a warning logged.
+
+    Raises
+    ------
+    FileNotFoundError
+        If `releases_dir` does not exist or is not a directory.
+    PermissionError
+        If there are permissions issues accessing `releases_dir` or its subdirectories.
+
+    Example
+    -------
+    >>> build_release_notes_with_brassy('/path/to/releases',
+    >>>                                 'https://example.com/license')
+
+    """
+    release_dirs = filter(
+        os.path.isdir,
+        [os.path.join(releases_dir, rd) for rd in os.listdir(releases_dir)],
+    )
+
+    with tempfile.NamedTemporaryFile(mode="w") as header_file:
+        header_file.write(
+            "\n".join(
+                [
+                    ".. dropdown:: Distribution Statement",
+                    "  | This file is auto-generated. Please abide by the license",
+                    f"  | found at {license_url}.",
+                ]
+            )
+        )
+
+        for release_dir in release_dirs:
+            release_filename = release_dir + ".rst"
+            release_version = release_dir.replace(releases_dir + "/", "")
+            if release_version == "upcoming":
+                log.warning(
+                    f"Skipping release dir {release_dir}"
+                    "because it's version is 'upcoming'"
+                )
+            log.debug(f"Setting version of {release_filename} to {release_version}")
+            build_release_note_from_dir_with_brassy(
+                release_dir, release_filename, release_version, header_file.name
+            )
+            # TODO: pythonize and call directly; requires update to pink
```

Comment:
Assuming this will come in a future update?

Reply:
Yes! It'll come after I release a new version of pink